packages/modules/viewer-react/src/hooks/useFrontstages.tsx

```diff
@@ -73,6 +73,7 @@ export const useFrontstages = ({
         id: "DefaultFrontstage",
         usage: StageUsage.General,
         contentGroupProps: contentGroup,
+        hideToolSettings: true,
```

Comment:
you should not need to explicitly set this. users would have no way to override this